.github/labeler.yml

```
@@ -0,0 +1,20 @@
+documentation:
+  - docs/**/*
```

```suggestion
  - docs/**/*
  - netlify.toml
  - .markdownlint-cli2.yml
  - **/*.md
```

the last entry in the proposed list is questionable :) But in the end, everything which touches a markdown file is documentation.

Just a minor suggestion.

---

I don't fully agree with that suggestion TBH.. i would put the tooling config files under `ops`. the markdown files under `documentation` are fine for me